### PR TITLE
add Manuscript Number to indexing

### LINKED ISSUE #721: Index and make searchable manuscript number
see `<shelfLocator>` in https://purl.stanford.edu/cf386wt1778.mods for where this is currently.

- [x] Index metadata
- [x] Design fielded search UI
- [x] Support fielded search
- [x] Support "exact match" searches for manuscript number #778

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR is connected to #721 and it only adds the manuscript_number_ssim Solr field.

### CODE REVIEW

**Comment by jkeck** [comment]
Of of the questions I had about the acceptance criteria of these tickets is what does "and make searchable" mean?  If it just means that it's part of the field list that is searched under general keyword search, then that's fine.  If it means that it is available as an option in the search drop down, then we may either need to add that to this PR or split that out into a diff. ticket.

**Comment by drh-stanford** [comment]
Yeah, I think this PR is just going to be the indexing. The searchability may need design work, or at least a strategy on whether these Parker fields are available across the whole site. In other words, if I add a config search field then it'll be available in all exhibits:

```

**Comment by drh-stanford** [comment]
I changed this PR to not close the main ticket. I think we need to discuss more _how_ these fields are searchable and whether it's specific to the Parker exhibit. I'll comment on that in the main ticket. If I just add a config search field, it'll show up in all the exhibits as I understand it:

```
    config.add_search_field('manuscript_number') do |field|
      field.label = 'Manuscript Number'
      field.include_in_advanced_search = false
      field.solr_parameters = { defType: 'lucene' }
      field.solr_local_parameters = {
        df: 'manuscript_number_ssim'
      }
    end
```

File: spec/models/spotlight/dor/indexer_spec.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
+# rubocop: disable Metrics/BlockLength
```
Comment (jkeck) [inline spec/models/spotlight/dor/indexer_spec.rb]: What would you think about disabling this for the `spec` directory entirely?

Comment (camillevilla) [reply spec/models/spotlight/dor/indexer_spec.rb]: +1 that notion ☝️

Comment (drh-stanford) [reply spec/models/spotlight/dor/indexer_spec.rb]: Fine with me. The rubocop todo has a huge number for this cop already.

**Comment by drh-stanford** [comment]
Ok, I've made the changes to Rubocop's configuration. The PR is ready for review.

**Comment by camillevilla** [comment]
@drh-stanford this indexing work looks good to me. We can split off another implementation ticket from #721 once the design work in #743 has been reviewed.

**Pull request merged**